sim/kubernetes/engine.go

```diff
@@ -231,7 +231,7 @@ func (kd *kubeEngine) CreateDeployment() (watch.Interface, error) {
 	// to have started.
 	w, err := kd.client.CoreV1().Pods(kd.namespace).Watch(opts)
 	if err != nil {
-		return nil, err
+		return nil, xerrors.Errorf("Pods could not be watched: %w", err)
```

Comment:
[Error string should not be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#error-strings).

Reply:
Same bellow.